ledger/acctonline.go

```diff
@@ -137,8 +137,8 @@ func (ao *onlineAccounts) loadFromDisk(l ledgerForTracker, lastBalancesRound bas
 	}
 
 	// the votes have a special dependency on the online accounts, so we need to initialize these separately.
-	ao.voters = &votersTracker{}
-	err = ao.voters.loadFromDisk(l, ao)
+	ao.voters = &votersTracker{onlineTopFunction: ao.onlineTop}
```

Comment:
I think it is so weird to be passing function pointers to struct methods like this. I know this came from the ledgercore refactor originally ... but why not make a one-method interface with the OnlineTop method in it and pass ao as an implementation of it?